containers/hub.Containerfile

```diff
@@ -0,0 +1,78 @@
+# FROM quay.io/centos/centos:stream8
+# FROM quay.io/sclorg/httpd-24-c8s
+# Builds should be available on https://quay.io/organization/openscanhub-fedora-infra
+FROM registry.access.redhat.com/ubi9/httpd-24
+USER 0
+
+RUN dnf install -y dnf-plugins-core https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm
+# RUN dnf config-manager --set-enabled crb
+
+RUN dnf -y --setopt=tsflags=nodocs install \
+    koji \
+    python3
+
+### END OF COMMON PART
+
+# enable installation of gettext message objects
+RUN rm /etc/rpm/macros.image-language-conf
+
+RUN dnf -y --setopt=tsflags=nodocs install \
+    /usr/bin/pg_isready \
+    csdiff \
+    gzip \
+    python3-bugzilla \
+    python3-csdiff \
+    python3-django3 \
+    python3-gssapi \
+    python3-jira \
+    python3-psycopg2 \
+    python3-qpid-proton \
+    xz
```

Comment:
These are all dependencies of `osh-hub`.  Why do we need to install them manually?

Reply:
This is still a draft and I need to clean it up before its ready to be merged.